scripts/prepare_data.py

```diff
@@ -0,0 +1,222 @@
+import json
+import pandas as pd
+import numpy as np
+
+from abc import ABC, abstractmethod, abstractproperty
+from pathlib import Path
+from django_countries import countries
+
+
+def meter_from_feet(feet):
+    """Converts feet to meters.
+
+    :param feet: value in feet
+    :type feet: float
+    :returns: value in meters
+    :rtype: float
+    """
+    ratio = 1 / 3.28084
+    return feet * ratio
+
+
+def country_code_by_name(name):
+    """Converts given country name to corresponding ISO-3166 alpha-2 country code.
+
+    :param name: human readable country name
+    :type name: str
+    :returns: ISO-3166 alpha-2 country code
+    :rtype: str
+    """
+
+    if name is None:
+        return None
+    elif name == 'United States':
+        return 'US'
+    elif name == "Cote d'Ivoire":
```

Comment:
Personally I like breaking the consistency of using `'` when I can ommit escaping in string, but according to our (google) rules this should be:
`'Cote d\'Ivoire'`
Just so you know ;)